### PR TITLE
Updating the Xtend community website

### PR DESCRIPTION
**Pull request by miklossy**

### CODE REVIEW

**Comment by miklossy** [comment]
**For reviewers:** this PR contains 3 commits to be more comfortable to review and to provide a more readable history.

File: xtend-website/community.html [hunk 1]
```diff
@@ -5,114 +5,121 @@
...
+							<div class="span4">
+								<h3>Discussion Group</h3>
+							<p></p>
+						</div>
 					</div>
-				    <div class="span4">
-				      <h3 >Found a Bug?</h3>
+				</div>
+				<div class="span5 team">
+					<a href="https://github.com/eclipse/xtext-xtend/issues" class="anchor-in-div" title="Bug reports and enhancement request are tracked at GitHub. Please explain the problem and provide a reduced but reproducable eample. Also please explain the concrete use case when requesting enhancements."></a>
```
Comment (tivervac) [inline xtend-website/community.html:22]: s/at Github/on Github/

s/eample/example/

File: xtend-website/community.html [hunk 2]
```diff
@@ -5,114 +5,121 @@
...
-			  </div>
-			</div>
-			<div class="span5 team">
-				 <a href="https://github.com/eclipse/xtext-xtend/issues" class="anchor-in-div" title="Bug reports and enhancement request are tracked at GitHub. Please explain the problem and provide a reduced but reproducable eample. Also please explain the concrete use case when requesting enhancements."></a>
-				 <div class="row">
-				    <div class="span1">
-				      <img src="{{ site.baseurl }}/images/bug-white.png" width="80" height="80" alt="image" class="image_left">
+			<div class="row">
+				<div class="span5 team">
+					<a href="https://groups.google.com/group/xtend-lang" class="anchor-in-div" title="The Xtend Google Group is the first source for getting answers in case you got stuck. The community is very friendly. Please ask any questions clearly and with sufficient information attached."></a>
```
Comment (tivervac) [inline xtend-website/community.html:10]: Wasn't gitter the first source?

File: xtend-website/community.html (hunk 1 above)
Comment (miklossy) [reply xtend-website/community.html]: Fixed.

File: xtend-website/community.html (hunk 2 above)
Comment (miklossy) [reply xtend-website/community.html:10]: I do not know.

Comment (cdietrich) [reply xtend-website/community.html:10]: nobody uses that group anymore. its just used from spam.
i'd prefer anything like the forum or stackoverflow over the d*mn google group

Comment (miklossy) [reply xtend-website/community.html:10]: @cdietrich : If you provide the stackoverflow/forum links as comment I will update the PR.

Comment (cdietrich) [reply xtend-website/community.html:10]: ?
https://www.eclipse.org/forums/index.php/f/27/
https://stackoverflow.com/

Comment (miklossy) [reply xtend-website/community.html:10]: Thanks. I was not sure whether there is dedicated channels to Xtend or the Xtext channels should be used.

Comment (miklossy) [reply xtend-website/community.html]: Done. @cdietrich Please review.

**Pull request merged**